quickquickstarts/Using_Validators_with_a_Linux_image/validators_with_a_linux_image.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Please keep the filename consistent with existing naming scheme.
If current scheme doesn't seem okay to you then please change the scheme for all of them - but the customer should have a consistent experience.